tests/system.py

```diff
@@ -829,6 +832,58 @@ def test_delete_column_family(self):
         # Make sure we have successfully deleted it.
         self.assertEqual(temp_table.list_column_families(), {})
 
+    def test_backup(self):
+        temp_table_id = "test-backup-table"
+        temp_table = Config.INSTANCE_DATA.table(temp_table_id)
+        temp_table.create()
+        self.tables_to_delete.append(temp_table)
+
+        temp_backup_id = "test-backup"
+        expire = int(datetime.datetime.now().timestamp()) + 604800
+
+        # Testing `Table.backup()` factory
+        temp_backup = temp_table.backup(
+            temp_backup_id,
+            cluster_id=CLUSTER_ID_DATA,
+            expire_time=datetime.datetime.utcfromtimestamp(expire),
+        )
+
+        # Sanity check for `Backup.exists()` method
+        self.assertFalse(temp_backup.exists())
+
+        # Tesing `Backup.create()` method
+        _ = temp_backup.create().result()
```

Comment:
I might be missing it, but it doesn't seem we use `_`. What is the value in assigning it then?

Reply:
Sorry, old habits) Thanks for catching!